uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/utils/JDK8Check.java

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc. 
+ * Distributed under license by Red Hat, Inc. All rights reserved. 
+ * This program is made available under the terms of the 
+ * Eclipse Public License v1.0 which accompanies this distribution, 
+ * and is available at https://www.eclipse.org/legal/epl-v10.html 
+ * 
+ * Contributors: 
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.fuse.reddeer.utils;
+
+import org.eclipse.reddeer.common.condition.WaitCondition;
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitUntil;
+import org.eclipse.reddeer.swt.condition.ShellIsAvailable;
+import org.eclipse.reddeer.swt.impl.button.OkButton;
+import org.eclipse.reddeer.swt.impl.shell.DefaultShell;
+import org.eclipse.reddeer.workbench.ui.dialogs.WorkbenchPreferenceDialog;
+import org.jboss.tools.fuse.reddeer.preference.InstalledJREs;
+
+/**
+ * Checks, if JDK8 is available.
+ * 
+ * @author djelinek, fpospisi
+ */
+public class JDK8Check {
+
+	public static final String JDK_WARNING_MESSAGE = "No Strictly compliant JRE detected";
+
+	/*
+	 * Checks, if JDK8 is available in instance of CRS.
```

Comment:
CRS means CodeReady Studio?

This code is also used in Fuse Tooling only. What about avoiding the acronym and replace with a more generic indication?